airgun/entities/settings.py

```diff
@@ -26,6 +26,11 @@ def update(self, property_name, value):
         view.validations.assert_no_errors()
         view.wait_for_update()
 
+    def contents(self):
+        """ Read the contents of setting browser"""
+        view = SettingsView(self.browser)
+        return view
+
```

Comment:
In Airgun we have a strategy that all lower-level activity is done in Airgun, In that case you could use the function as send_email instead return the view.

Reply:
Agree, But to check the status of sent email notificaton, contents of view is required.
  ```
  def sendmail(self):
        """ Read the contents of setting browser"""
        view = SettingsView(self.browser)
        view.email_button.click()
        return view
```

Reply:
that's fine